webapp/test/utils/sortTokens.test.ts

```diff
@@ -0,0 +1,200 @@
+import { StakeToken } from 'types/stake'
+import { sortTokens } from 'utils/sortTokens'
+import { describe, expect, it } from 'vitest'
+
+const createMockToken = (
+  symbol: string,
+  balance?: bigint,
+  address = `0x${symbol.toLowerCase().padEnd(40, '0')}`,
+  chainId = 43111,
+  decimals = 18,
+  // eslint-disable-next-line max-params
+): StakeToken => ({
+  address,
+  balance,
+  chainId,
+  decimals,
+  extensions: {
+    protocol: 'hemi',
+    rewards: [],
+    website: 'https://example.com',
+  },
+  logoURI: '',
+  name: symbol,
+  symbol,
+})
```

Comment:
```suggestion
const createMockToken = function (params: {
  symbol: string,
  balance?: bigint,
  address,
  chainId
  decimals
): StakeToken {
  const { address = zeroAddress, balance,  chainId = 43111,   decimals = 18, symbol } = params ?? {}
  return {
    address,
    balance,
    chainId,
    decimals,
    extensions: {
      protocol: 'hemi',
      rewards: [],
      website: 'https://example.com',
    },
    logoURI: '',
    name: symbol,
    symbol,
  }
}
```


with this you can just call

```ts
createMockToken({ symbol: 'DAI', balance: BigInt(1000000) })
// this is also valid, it's easier to infer what each parameter does , and no longer depends on the order of the parameters
createMockToken({ balance: BigInt(1000000), symbol: 'DAI',  })
```


Note that I imported `zeroAddress` from `viem`

Reply:
@gndelia Noted, Resolved!